docs/explanation/pebble.rst

```diff
@@ -0,0 +1,33 @@
+.. _pebble_explanation_page:
+
+Pebble
+======
+
+When creating container images using Dockerfile, several issues can be met.
```

Comment:
Perhaps we can start with saying what Pebble is:

> Pebble is a service manager that helps to organise a set of local service processes.

Reply:
done :)